### PR TITLE
Change Of Password Done

### PR DESCRIPTION
**Pull request by java80**
Please take a look and merge

### CODE REVIEW

**Review by foremanlb — COMMENTED** [review]
functionally looks good to me, can someone check for linting as that is def not my expertise.

File: controllers/tutors.js [hunk 1]
```diff
@@ -69,4 +69,34 @@ const signIn = async (req, res) => {
...
+      const password_digest = await bcrypt.hash(newPassword, SALT_ROUNDS);
+      tutor = await Tutor.findByIdAndUpdate(
+        req.params.id,
+        { password_digest: password_digest },
+        { new: true }
+      );
+      const payload = {
+        id: tutor._id,
+        username: tutor.username,
+        email: user.email,
```
Comment (GundamMerlin) [inline controllers/tutors.js]: change to tutor.email,

File: controllers/tutors.js [hunk 2]
```diff
@@ -69,4 +69,34 @@ const signIn = async (req, res) => {
...
+      };
+      const token = jwt.sign(payload, TOKEN_KEY);
+      return res.status(201).json({ user, token });
+    }
+    else {
+      return res.status(403).send("wrong password")
```
Comment (GundamMerlin) [inline controllers/tutors.js]: Not super necessary but maybe change the message to "Invalid Password"

Comment (java80) [reply controllers/tutors.js]: I will make the changes

File: routes/tutors.js [hunk 3]
```diff
@@ -1,3 +1,12 @@
 const { Router } = require('express')
+const { signUp } = require("../controllers/tutors.js")
```
Comment (foremanlb) [inline routes/tutors.js]: please add the rest of the imports to coincide with the functions

File: routes/tutors.js [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+const restrict = require("../helpers/restrict.js")
 
 const router = Router()
+router.put("/password-change/:id,changePassword");
```
Comment (foremanlb) [inline routes/tutors.js]: changePassword should not be in quotes

**Review by foremanlb — APPROVED** [review]

**Pull request merged**